app/src/main/java/com/minimalisticapps/priceconverter/presentation/ui/item/ItemFiatCoin.kt

```diff
@@ -69,15 +69,15 @@ fun ItemFiatCoin(
                 "content description",
                 modifier = Modifier
                     .padding(start = 0.dp, end = 15.dp)
-                    .clickable { onDeleteClick(bitPayCoinWithFiatCoin.fiatCoinExchange) }
+                    .clickable { onDeleteClick(index) }
             )
         }
-        val btcValue =
-            bitPayCoinWithFiatCoin.bitPayExchangeRate.oneShitCoinValueString
+        val unitOfShitcoinInBtcFormatted =
+            formatUnitOfShitcoinPrice(oneUnitOfShitcoinInBTC)
```

Comment:
Component is responsible for formatting. Originally the formatted value was in the model. That was violating Single Responsibility Principle.